### PR TITLE
SHS-5581: Remove Chosen plugin dependency

### PR DESCRIPTION
**Pull request by codechefmarc**
# READY FOR REVIEW

## Summary
* Uninstalls `chosen_lib` from Drupal
* Removes chosen CSS class references
* Removes chosen JS references
* Does not remove chosen from `composer.json` as that will take place in a subsequent PR

## Need Review By (Date)
2024-05-16

## Urgency
low

## Steps to Test
1. Login to Drupal and visit `/admin/modules`
2. Verify that Chosen and Chosen Library are both uninstalled
3. Visit a page that has a multi-select box, such as Political Science (`/undergraduate-program/current-courses`), Creative Writing (`/books`), or HS Colorful (`/default-views/courses`)
4. Verify that the multi-select box still works with no visual changes

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_airy/css/vendor.we-megamenu-base.css [hunk 1]
```diff
@@ -1,7 +1,6 @@
 // We Megamenu CSS Imported from module
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_airy/css/vendor.we-megamenu-base.css]: This and other vendor megamenu files are copied straight from the Drupal Megamenu library that was used on the site before and was replaced by a custom implementation. We should probably explore in the future removing or cleaning up these files, but for now lets leave them untouched. Please revert them to their original version.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@codechefmarc Please check my comment on the megamenu vendor CSS. Other than that, looks great, thanks!

File: docroot/themes/humsci/humsci_airy/css/vendor.we-megamenu-base.css (hunk 1 above)
Comment (codechefmarc) [reply docroot/themes/humsci/humsci_airy/css/vendor.we-megamenu-base.css]: Ok, reverted those files. Please review. Thank you!

**Review by cienvaras — COMMENTED** [review]
Works great, thanks! 🚀

**Pull request merged**